stories/topics/tech-note-gcp-upgrade.adoc

```diff
@@ -0,0 +1,5 @@
+[id="tech-note-gcp-upgrade"]
+
+= Upgrade note
+
+Upgrading {PlatformNameShort} to 2.4.20230630 will update the protocol of its internal load balancers from HTTP to HTTPs. If additional networking configurations were added, they may also need to be updated to ensure connectivity. After the upgrade has suceeded, it is recommended to revalidate additional added networking configurations.
```

Comment:
Upgrading {PlatformNameShort} to 2.4.20230630 updates the protocol...

they can also be updated....

When the upgrade has succeeded, you must revalidate any...